tests/phpunit/BltProject/FixTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Acquia\Blt\Tests\BltProject;
+
+use Acquia\Blt\Tests\BltProjectTestBase;
+
+/**
+ * Class FixTasksTest.
+ *
+ * Verifies that fix related tasks work as expected.
+ */
+class FixTasksTest extends BltProjectTestBase {
+
+  private $returnVar;
+  private $output = [];
+
+  /**
+   * Tests for existence of phpcbf script.
+   */
+  public function testFixPhpcbfExists() {
+    $this->executePhpcbfFileListScript();
+    $this->assertNotContains("PHP Code Beautifier was not found in this project's bin directory. Please run composer install.", $this->output);
+    $this->assertEquals(1, $this->returnVar);
+  }
+
+  /**
+   * Tests the handling of a missing file parameter.
+   */
+  public function testFixPhpcbfExitsWhenMissingArguments() {
+    $this->executePhpcbfFileListScript();
+    $this->assertContains("Missing file list parameter.", $this->output);
+    $this->assertEquals(1, $this->returnVar);
+  }
+
+  /**
+   * Tests that a file list is generated.
+   */
+  public function testFixPhpcbfWithArguments() {
+    $report = $this->createTestReport();
+    $this->executePhpcbfFileListScript($report);
+    $this->assertNotContains("Missing file list parameter.", $this->output);
+    $this->assertContains("Files that can be fixed by PHPCBF:", $this->output);
+    $this->assertContains("{$this->drupalRoot}/foo.php", $this->output);
+    $this->assertEquals(0, $this->returnVar);
+  }
+
+  /**
+   * Helper method that xecutes the phpcbf-file-list.sh script.
+   *
+   * @param string $file
+   *   The full path to the file to be processed.
+   */
+  protected function executePhpcbfFileListScript($file = '') {
+    chdir($this->projectDirectory);
+    $command = "sh vendor/acquia/blt/scripts/blt/phpcbf-file-list.sh $file";
+    exec($command, $this->output, $this->returnVar);
+  }
+
+  /**
+   * Helper method to create a temp file, formatted as a phpcs csv report.
+   */
+  protected function createTestReport() {
+    $tmp_file = tempnam(sys_get_temp_dir(), '');
+    if (!$tmp_file) {
+      throw new \Exception("Unable to create temporary file.");
+    }
+    $file_contents = "\"{$this->projectDirectory}/docroot/foo.php\",1,1,error,\"Line indented incorrectly; expected 2 spaces, found 4\",Drupal.WhiteSpace.ScopeIndent.IncorrectExact,5,1";
+    $command = "echo '$file_contents' > {$tmp_file}";
```

Comment:
@dooleymatt [`file_put_contents()`](http://php.net/manual/en/function.file-put-contents.php) would be simpler, since we're writing PHP. (I realize I put you on the wrong scent by using the word "echo".)